src/components/SessionListItem/SessionListItem.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import style from './SessionListItem.module.scss';
+
+const SessionListItem = ({session}) => {
+    const [isSelected, setIsSelected] = useState(false);
+
+const background = session.isTeaching ? style.teach : style.learn;
+
+    return(
+        <div className = {style.wrapper + ' ' + background}> 
+            <div className={style.info}>
+                <span className={style.status}>
+                    {
+                        session.isTeaching ? 'Teaching' : 'Learning'
+                    }
+                </span>
+                <span>{`${session.subject} with ${session.name}`}</span>
+            </div>
+            <div className={style.info}>
+                <a className={style.link} 
+                    onClick={() => setIsSelected(!isSelected)}
```

Comment:
Same for this onclick part here, can be removed, but keep the link just not the onClick part